app/views/corrections/VatPeriodCorrectionsListView.scala.html

```diff
@@ -37,15 +37,34 @@
 
     @layout(pageTitle = dynamicTitle ) {
 
-        @formHelper(action = controllers.corrections.routes.VatPeriodCorrectionsListController.onSubmit(mode, period), 'autoComplete -> "off") {
+        <header class="hmrc-page-heading">
+            <h1 class="govuk-heading-xl">@dynamicTitle</h1>
+        </header>
 
-            <header class="hmrc-page-heading">
-                <h1 class="govuk-heading-xl">@dynamicTitle</h1>
-            </header>
+        @if(incompletePeriods.nonEmpty) {
+        <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+                <span class="govuk-warning-text__assistive">Warning</span>
+                @messages("error.missing_answers")
+            </strong>
+        </div>
+        }
 
-            @addToList(periodList, itemSize = Long, "vatPeriodCorrectionsList.change.hidden", "vatPeriodCorrectionsList.remove.hidden")
+        @addToList(periodList, itemSize = Long, "vatPeriodCorrectionsList.change.hidden", "vatPeriodCorrectionsList.remove.hidden")
 
-            @button("site.continue",  period, controllers.corrections.routes.VatPeriodCorrectionsListController.onPageLoad(mode, period).url)
+        @if(incompletePeriods.isEmpty) {
+            @formHelper(action = controllers.corrections.routes.VatPeriodCorrectionsListController.onSubmit(mode, period, false), 'autoComplete -> "off") {
```

Comment:
I have noticed the setting of 'autoComplete' -> "off" in a lot of places I assume this is intentional, but what is the reason behind it?